### DISCUSSION TITLE
Electrical quantity management

### DISCUSSION THREAD

**Discussion by rickykung** [milo526/homebridge-tuya-web#204] [Ideas]
Some switch models supports showing statistic and electrical quantity. 
Is there exist this function in this plugin?
Or will it be the upcoming feature?

Thanks,
Ricky

**Comment by milo526** [comment]
Hello, sadly this is not a feature and will never be.
The API that this plugin uses does not expose this data.
Since I am not affiliated with Tuya, I cannot change their API and thus not show this information in the plugin.

**Reply by rickykung** [reply]
Thanks for the quick reply. Nvm. Well done for this plugin. Good day!
